### ISSUE TITLE
about how to test

### ISSUE THREAD

**Issue by zhj123169** [kenshohara/3D-ResNets-PyTorch#87]
Thank you for your great work！
I‘m a beginner of this area so part of this project seems difficult for me.For example,I want to test UCF101 to get accuracy using one of your Pre-trained models.How to make it???May I get the detailed method of testing，including parameters？
Looking forward to your reply！

**Comment by chowkamlee81** [comment]
hai..iam facing the same issue. Can you kindly help..

**Comment by akashp1196** [comment]
Any update on this? I've enabled the "--test" flag as well as "--test_subset test" for main.py but not having much luck testing.

**Comment by slighting666** [comment]
@akashp1196
Do you know how to run the code and load the required parameters before testing UCF101

**Comment by 87003697** [comment]
Sorry I took this repository as a referrence for my own code. You'd better ask kenshohara for details.



---Original---

**Comment by slighting666** [comment]
Can you give me some advice? My qq114974577







------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"ZhiyuanthePony"<notifications@github.com&gt;;
发送时间:&nbsp;2020年1月7日(星期二) 下午2:16
收件人:&nbsp;"kenshohara/3D-ResNets-PyTorch"<3D-ResNets-PyTorch@noreply.github.com&gt;;
抄送:&nbsp;"slighting"<1149745774@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [kenshohara/3D-ResNets-PyTorch] about how to test (#87)



Sorry I took this repository as a referrence for my own code. You'd better ask kenshohara for details. 
 
 
 
 ---Original---

**Comment by 87003697** [comment]
要不你加我吧，加你需要回答问题，我QQ870036797



---Original---

**Comment by HuangZuShu** [comment]
Do you know the difference between test(inference) and validation in ucf101，a little confused . I can't understand why we can use the same video in validation and test.@87003697

**Comment by whitepagewu** [comment]
After training. How to test using my trained model
